bluemix/terminal/ui.go

```diff
@@ -8,17 +8,27 @@ import (
 	. "github.com/IBM-Cloud/ibm-cloud-cli-sdk/i18n"
 )
 
+// UI provides utilities to handle input and output streams
 type UI interface {
-	// Say prints the formated message
+	// Deprecated: this method could be removed in the future,
+	// Use Info() if the message can be suppressed in quiet mode
+	// Or use Print() if the message will be printed anyway
+	// Say prints the formated message, the message will be suppressed in quiet mode
 	Say(format string, args ...interface{})
 
-	// Warn prints the formated warning message
+	// Info prints the formated failure message, the message will be suppressed in quiet mode
+	Info(format string, args ...interface{})
+
```

Comment:
If the method is to print the failure message, then it should be named as Error() instead of Info()